README.md

```diff
@@ -21,7 +21,7 @@ Most things that you can do manually in the browser can be done using Puppeteer!
 Here are a few examples to get you started:
 
 - Generate screenshots and PDFs of pages.
-- Crawl a SPA (Single-Page Application) and generate pre-rendered content (i.e.
+- Crawl an SPA (Single-Page Application) and generate pre-rendered content (i.e.
```

Comment:
This is incorrect a prefix for SPA should be `a` as the first letter is not a vowel. Please remove this change.

Reply:
No. For words like "studio" we say "*a* studio". For the "s" letter we say "**an** SUV".
  - https://oneminuteenglish.org/en/an-s-or-a-s-which-is-correct/
  - https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an

Reply:
Yes you are correct there are exceptions but I have only heard it pronounce like the word `spa` [`/spɑː/` (https://dictionary.cambridge.org/dictionary/english/spa?q=SPA)] with a constant start.  
The `SUV` has a pronunciation of `/ˌes.juːˈvi/` which has a vowel start.

Reply:
[MDN uses "**an** SPA"](https://developer.mozilla.org/en-US/docs/Glossary/SPA#:~:text=An%20SPA%20(Single%2Dpage%20application)).